### PR TITLE
DOC: update the docstring pandas.Series.dt.is_month_start

### PR DESCRIPTION
**Pull request by janelle**
Checklist for the pandas documentation sprint (ignore this if you are doing
an unrelated PR):

- [X ] PR title is "DOC: update the <your-function-or-method> docstring"
- [X] The validation script passes: `scripts/validate_docstrings.py <your-function-or-method>`
- [X] The PEP8 style check passes: `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [X] The html version looks good: `python doc/make.py --single <your-function-or-method>`
- [X] It has been proofread on language by another sprint participant

Please include the output of the validation script below between the "```" ticks:

```
################################################################################
################# Docstring (pandas.Series.dt.is_month_start)  #################
################################################################################

Return a boolean indicating if the date is the first day of the month.

Returns
-------
is_month_start : Series of boolean

See Also
--------
is_month_end : Returns a boolean indicating if the date is the last day
               of the month.

Examples
--------
>>> dates = pd.Series(pd.date_range("2018-02-27", periods=3))
>>> dates
0   2018-02-27
1   2018-02-28
2   2018-03-01
dtype: datetime64[ns]
>>> dates.dt.is_month_start
0    False
1    False
2    True
dtype: bool

################################################################################
################################## Validation ##################################
################################################################################

Errors found:
	No extended summary found

```

If the validation script still gives errors, but you think there is a good reason
to deviate in this case (and there are certainly such cases), please state this
explicitly.

Extended summary not required in this case (confirmed by peers).

### CODE REVIEW

**Comment by cemsbr** [comment]
Please, add "DOC: " prefix to the PR.

File: pandas/core/indexes/datetimes.py
Comment (cemsbr) [inline pandas/core/indexes/datetimes.py]: Full stop missing.

File: pandas/core/indexes/datetimes.py
Comment (cemsbr) [inline pandas/core/indexes/datetimes.py]: Please, change to infinitive "Return". "if" should be "whether".

**Comment by TomAugspurger** [comment]
This LGTM once you fix the line to long issues. Ping me once that's fixed and we'll merge it.

File: pandas/core/indexes/datetimes.py
Comment (cemsbr) [inline pandas/core/indexes/datetimes.py]: Missing full-stop.

File: pandas/core/indexes/datetimes.py
Comment (cemsbr) [inline pandas/core/indexes/datetimes.py]: As stated before: please, change to infinitive "Return". "if" should be "whether".

**Review by cemsbr — CHANGES_REQUESTED** [review]

**Comment by janelle** [comment]
closing as I might have mixed up two branches by accident, sorry
will open again for pandas.Series.dt.is_month_last to avoid confusion

**Pull request closed without merging**
